maml_rl/baseline.py

```diff
@@ -55,19 +55,27 @@ def _feature(self, episodes):
     def _tw_feature(self, episodes):
         ep_obs_str = episodes.observations
         ep_triplets = episodes.triplets
+        ep_lengths = episodes.lengths
+        max_ep_length = len(episodes)
         ep_h_og = []
         ep_obs_mask = []
         ep_h_go = []
         ep_node_mask = []
         for i in range(episodes.batch_size):
             h_og, obs_mask, h_go, node_mask = self.agent.encode(ep_obs_str[i], ep_triplets[i], use_model="policy")
             #ep_h_og.append(h_og.unsqueeze(0))
+
+            h_go = F.pad(h_go, (0, 0, 0, 0, 0, max_ep_length-ep_lengths[i])) # hardcoded not only for graph input--but for padding too--suggestions?
```

Comment:
don't like this hardcoding. essentially i _need_ to know shape of the tensors to pad it.